### ISSUE TITLE
datachart-axis-label-templates.adoc

### ISSUE THREAD

**Issue by pamelabrasil** [Infragistics/convertwire#20]
Link inserted in control name:
xamData%%ApiLink%%.%%ControlsName%%.htmlChart

DocX GUID: c32b5e51-5bf7-40cd-861b-38a5c2f83208

**Comment by craigshoemaker** [comment]
:thought_balloon: no idea how that showed up there. I'll dig into it...

**Comment by craigshoemaker** [comment]
This is in the original document, but since it's an empty tag, you can't tell. Here's what it looks like in the source document:

```
xamData<a href="%%ApiLink%%.%%ControlsName%%.html"></a>Chart control
```

I will look and see if I can write a RegEx to strip out all HTML tags that have no content inside them. If that doesn't work then this goes on the manual clean up list.

**Comment by ig-ealbert** [comment]
@craigshoemaker That's a dangerous solution as there are many internal links that are <a id="something"></a>.  Those should not be removed.  I would recommend that this be logged in TFS as a content bug and fixed through the XML file in TFS or through Document X.
This is all caused by the Word -> HTML conversion.  Word makes some very scary HTML, especially with the review process.

**Comment by craigshoemaker** [comment]
@ig-ealbert I don't see how removing empty tags is a dangerous action. Can you give me an example?

**Comment by ig-ealbert** [comment]
view-source:http://help.infragistics.com/Help/Doc/WPF/2015.2/CLR4.0/html/DataChart_Getting_Started_with_DataChart.html#Ref397366350
`<h4><a id="Ref397366350" name="Ref397366350"></a>Understanding Chart</h4>`

Any internal anchor tag will be "empty" so that when you click on the link in the "In This Topic" section, you will go to the place in the topic that corresponds to it.

**Comment by craigshoemaker** [comment]
closed by 2515d2a8c3ba4397969d9e172a3973a2b105b8d3

Check out the explanation I provide to #28 for a [full explanation](https://github.com/Infragistics/convertwire/issues/28#issuecomment-159278395)
